stable/kubecost/values.yaml

```diff
@@ -87,13 +87,19 @@ cost-analyzer:
       # This overrides the cluster_id set in prometheus.server.global.external_labels.
       # NOTE: This does not affect the external_labels set in prometheus config.
       clusterIDConfigmap: kubecost-cluster-info-configmap
+      image:
+        tag: v2.19.1
```

Comment:
I am just concerned with the version bump here since the kubecost team hasn't vetted their product with this version of prometheus so it's possible something may break if we upgrade the version. All the other changes lgtm

Reply:
Yeah, we have not tested against this version yet. Should be fine but can't say with a high degree of certainty until we run all of our tests against this. Is there a specific feature you need?

Reply:
nothing should break as it is only a minor version change and nothing was mentioned in the release notes.
https://github.com/prometheus/prometheus/releases
enhancements, fixes
some related to memory consumption

Reply:
We are running version 2.19.0 in soak as of yesterday to clear up a query error.